Makefile

```diff
@@ -304,6 +322,31 @@ release-alias-tag: ## Add the release alias tag to the last build tag
 release-templates: $(RELEASE_DIR) ## Generate release templates
 	cp templates/cluster-template*.yaml $(RELEASE_DIR)/
 
+.PHONY: release-binaries
+release-binaries: ## Builds the binaries to publish with a release
```

Comment:
and also we need to call this target in the release target so that we generate this artefact as well when we release

Reply:
and we also need to explore how to ship it for the pre-GA builds

Reply:
Fixed.